src/components/cards/PwcCard.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+    <card v-if="showPWCSchedule" class="card">
+        <p class="title">PWC Hours</p>
+        <div class="green-line"></div>
+        <div class="countdown">
+            <p>{{ openingStatus }}</p>
+            <p class="time-text">{{ timeStatus }}</p>
+        </div>
+    </card>
+</template>
+
+<script>
+import { mapState } from 'pinia';
+import Card from '@/components/Card.vue';
+import useUserSettingsStore from '@/stores/user-settings';
+
+export default {
+  components: { Card },
+  data() {
+    return {
+      openingTimes: [
+        { day: 1, start: '15:30', end: '20:00' }, // Monday
+        { day: 2, start: '15:30', end: '20:00' }, // Tuesday
+        { day: 3, start: '15:30', end: '20:00' }, // Wednesday
+        { day: 4, start: '15:30', end: '20:00' }, // Thursday
+        { day: 5, start: '15:30', end: '18:00' }, // Friday
+        { day: 6, start: '10:00', end: '15:00' }, // Saturday
+        { day: 0, start: '10:00', end: '15:00' }, // Sunday
+      ],
+      currentTimeMs: 0,
```

Comment:
so we can get rid of this along with the interval. there is already a variable that gets updated every second in our state management system (pinia).

`    ...mapState(useScheduleStore, ['date']),
`

Reply:
When I deleted this line the countdown stops updating every second and becomes static for some reason :(